packages/odie-client/src/components/message/user-message.tsx

```diff
@@ -42,13 +31,20 @@ export const UserMessage = ( {
 	} = useOdieAssistantContext();
 
 	const hasCannedResponse = message.context?.flags?.canned_response;
-	const isRequestingHumanSupport = message.context?.flags?.forward_to_human_support;
+	const isRequestingHumanSupport = message.context?.flags?.forward_to_human_support ?? false;
 	const hasFeedback = !! message?.rating_value;
 	const isBot = message.role === 'bot';
 	const isPositiveFeedback =
 		hasFeedback && message && message.rating_value && +message.rating_value === 1;
-	const showExtraContactOptions =
-		hasFeedback && ! isPositiveFeedback && userHasEverEscalatedToHumanSupport;
+	const isConnectedToZendesk = chat?.provider === 'zendesk';
+
+	let showExtraContactOptions: boolean;
+
+	if ( experimentName === 'give_a_chance' ) {
```

Comment:
Same on experiment name. Since this is called in multiple places, perhaps, it's best to centralize in one place.

We should also have to be aware that the entire logic will fall apart if we stop the experiment and rollout to production. I don't have a solution other than "let's be very aware that we'll have to change this when the experiment ends". Perhaps centralizing into a flag will make this job easier.